File: CppPractice/CppLeetcode/Leetcode26.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int slow = 0;
        int ans= 1;
        if(nums.size()==1){return 1;}
        for(int i=0;i<nums.size();i++)
        {
            if(nums[i]!=nums[slow]){slow++;nums[slow]=nums[i];ans++;}
        }
        return ans;
    }
};

File: CSharpPractice/LeetcodePractice/Leetcode26.cs
public class Solution26 {
    public int RemoveDuplicates(int[] nums) 
    {
        int slow = 0;
        for(int i=1; i< nums.Length;i++)
        {
            if (nums[i] != nums[slow])
            {
                slow++;
                nums[slow] = nums[i];
            }
        }
        return slow+1;
    }
}